### ISSUE TITLE
Posts page

### ISSUE THREAD

**Issue by schalkneethling** [mdn/developer-portal#1722]
A couple of small items that need attention:

1. [ ] The main content should be marked up as such.

```
<main id="results-list" role="main">
```

as opposed to:

```
<div id="results-list">
```

2. The posts lists should ideally be marked up as a list:

```
<div class="mzp-l-main custom-width">
  <ul>
    <li mzp-c-card mzp-c-card-medium mzp-has-aspect-16-9">
      <a class="mzp-c-card-block-link" href="https://mozillagfx.wordpress.com/2020/07/10/moz-gfx-newsletter-54/" target="_blank" rel="noopener noreferrer" data-type="article">
        ...
      </a>
    </li>
  </ul>
</div>
```

**Comment by stevejalim** [comment]
List markup is handled in work done for #1723
